cms/templates/certificates.html

```diff
@@ -30,7 +30,9 @@
 
 <%block name="requirejs">
   require(["js/certificates/factories/certificates_page_factory"], function(CertificatesPageFactory) {
-      CertificatesPageFactory(${escape_json_dumps(certificates) | n}, "${certificate_url}", "${course_outline_url}", ${escape_json_dumps(course_modes) | n}, ${escape_json_dumps(certificate_web_view_url) | n}, ${escape_json_dumps(is_active) | n}, ${escape_json_dumps(certificate_activation_handler_url) | n} );
+      if('${has_certificate_modes}'=='True' ? true : false;) {
```

Comment:
As a nit, I'd rather see this like so:
```
if (${escape_json_dumps(has_certificate_modes)}) {
    // stuff
}
```
Just to maintain the convention that we escape everything (unless there's a good reason not to) in templates.